frontend/src/lib/SegmentMetricsManager.ts

```diff
@@ -89,18 +89,6 @@ export class SegmentMetricsManager implements MetricsManager {
     if (this.actuallySendMetrics) {
       // Segment will not initialize if this is rendered with SSR
       initializeSegment()
-
-      const userTraits: any = {
-        ...this.getInstallationData(),
-        ...this.getHostTrackingData(),
-      }
-
-      // Only record the user's email if they entered a non-empty one.
-      if (this.sessionInfo.current.authorEmail !== "") {
-        userTraits.authoremail = this.sessionInfo.current.authorEmail
-      }
-
-      this.identify(this.sessionInfo.current.installationId, userTraits)
```

Comment:
It looks like we may no longer be using `SessionInfo.authorEmail` or `SessionInfo.installationId`, which means we should also remove these properties from the SessionInfo interface (and from the protobuf that sends them over from the server). 

(Doesn't necessarily have to be done in this PR, but we should do it shortly after this one lands.)

Reply:
Happy to try and remove authorEmail in this PR. We actually do send installationId separately [here](https://github.com/streamlit/streamlit/blob/64fe93b7f87d59c4d6fd28f8dfdcf64200666b22/frontend/src/lib/SegmentMetricsManager.ts#L165), so we should keep that here.

Reply:
Removed authorEmail from sessionInfo